stock_by_warehouse_mrp_bom/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+Provide a nice widget in the product on the mrp_bom line to see the quantity of product
+by warehouse in order to help you to review such information as fast as possible.
+
+.. image:: http://screenshots.vauxoo.com/nhomar/stock_by_warehouse_sale_1.png
+    :alt: Example on Form View
+    :width: 600px
+
+.. image:: http://screenshots.vauxoo.com/nhomar/stock_by_warehouse_sale_2.png
+    :alt: Example on Form View
+    :width: 600px
+
+Developers usage:
```

Comment:
We can delete this section, because this usage is for the `stock_by_warehouse` module

Reply:
This section still needs to be deleted 🙈